sdk/eventhubs/azure-messaging-eventhubs/src/eventhubs_utilities.cpp

```diff
@@ -0,0 +1,2 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Does this file need to exist ?

Reply:
Not today, but I suspect I'll be moving code into it in the future as I find more functions to add to eventhubs_utilities.